config/docker/set/command.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+import os
+from typing import List
+
+from dt_shell import DTCommandAbs, dtslogger, DTShell
+
+
+class DTCommand(DTCommandAbs):
+    help = "Configure docker registry credentials"
+
+    usage = f"""
+Usage:
+
+    dts config docker --docker-username <your-docker-username> --docker-password <your-docker-access-token> [--docker-registry <optional-specific-registry>]
```

Comment:
Replace `--docker-username` with `--username`, `--docker-password` with `--password`, and `--docker-registry` with an optional positional argument called `registry`.

Reply:
For the same reason, to resemble `docker login` ([format](https://docs.docker.com/engine/reference/commandline/login/)), I named the `registry` positional argument as `server` in the new commit